amoro-web/src/main/resources/static/assets/404-Bb4-4nNW.css

```diff
@@ -0,0 +1,20 @@
+
+/*
+  * Licensed to the Apache Software Foundation (ASF) under one
+  * or more contributor license agreements.  See the NOTICE file
+  * distributed with this work for additional information
+  * regarding copyright ownership.  The ASF licenses this file
+  * to you under the Apache License, Version 2.0 (the
+  * "License"); you may not use this file except in compliance
+  * with the License.  You may obtain a copy of the License at
+  *
+  *     http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
+
+.container[data-v-201aa400]{padding-top:100px}.container .title[data-v-201aa400]{font-size:20px;font-weight:700}.container .button[data-v-201aa400]{background:#1890ff;border-radius:3px;color:#fff}
```

Comment:
The files under `amoro-web/src/main/resources/static/`  should not be added to git

Reply:
check again please